### DISCUSSION TITLE
Fluxnews - Enhance Post Management Screen

### DISCUSSION THREAD

**Discussion by wildntender** [inspireui/support#5601] [New Feature/Ideas Requests]
Hi team,

I would like to request a feature to add an option to record/select video/audio from Post management screen in Fluxnews app.

![PostManagement](https://user-images.githubusercontent.com/112659461/187998814-a0fe6906-5d25-4e04-9d2c-bee5d3d27c34.png)

**Why this request:**

This will be helpful in a lots of ways and will definitely increase usage/sales of app by saving time, the news reaches almost instantly to the subscribers. 

User doesn't need full recording setup and can directly upload from phone. Also saves time wasted while waiting for you-tube approval.

1. Normal V-logger can directly record the page by clicking button on post management and post it directly on app. 

2. News reporter can record and upload directly from a news scene

3. For e.g. I am food blogger and going through a food market, I can just record and upload a good dish being made.

4. App author can live blog and upload to site, and subscribers can add comment on post and the user gets a feedback instantly.

**JetPack wordpress app** allows this feature :  https://jetpack.com/blog/jetpack-wordpress-app-manage-your-website/

![Jetpack](https://user-images.githubusercontent.com/112659461/187998659-b2af3275-f118-401c-b373-f4347dda521c.png)

**Comment by shailendersindhu** [comment]
my +1 as well for this feature, this will make posting fast content so much easier. @minhcasi
